Werehouse/Werehouse.py

```diff
@@ -0,0 +1,57 @@
+helmet = 0

```

Comment:
Ok looks good now - please move your project to path `makolele/main.py`, we can merge it and you can start next task :)

Reply:
You mean main/makolele/main.py?
Now it is in Makolele/main.py